### PR TITLE
Update project to v1.8.0

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

File: run-tests_android.sh [hunk 1]
```diff
@@ -27,9 +27,15 @@ echo "Installing pip for python"
 python -m venv .venv
```
Comment (rucindrea) [inline run-tests_android.sh]: this is not needed anymore, we are no longer using the virtual environemtn.

File: run-tests_android.sh [hunk 2]
```diff
@@ -27,9 +27,15 @@ echo "Installing pip for python"
...
 source .venv/bin/activate
```
Comment (rucindrea) [inline run-tests_android.sh]: same here, no longer needed

File: run-tests_android.sh [hunk 3]
```diff
@@ -42,7 +48,15 @@ export APPIUM_APPFILE=$PWD/application.apk #App file is at current working folde
 ## Run the test:
 echo "Running tests"
 
+# rm -rf screenshots
+# /test/.venv/bin/python -m pytest tests/ --junitxml=test-reports/report.xml
+
+# mv test-reports/*.xml TEST-all.xml
+## Run the test:
+
```
Comment (rucindrea) [inline run-tests_android.sh]: these seem to be commented and then used.

File: run-tests_android.sh [hunk 4]
```diff
@@ -42,7 +48,15 @@ export APPIUM_APPFILE=$PWD/application.apk #App file is at current working folde
...
 rm -rf screenshots
-python -m pytest tests/ --junitxml=test-reports/report.xml
+ls -la
+pwd
```
Comment (rucindrea) [inline run-tests_android.sh]: ls and pwd are probably not needed anymore?

File: run-tests_ios.sh [hunk 5]
```diff
@@ -10,10 +10,15 @@ echo "Setting up python env:"
 python3 -m venv .venv
 source .venv/bin/activate
```
Comment (rucindrea) [inline run-tests_ios.sh]: same here, the two lines above are not needed.

File: tests/game_play_test.py [hunk 6]
```diff
@@ -19,26 +19,24 @@ def setUp(self):
...
         self.pause_overlay_page.press_main_menu()
-        assert(self.main_menu_page.is_displayed())
+        assert self.main_menu_page.is_displayed()
 
     def test_avoiding_obstacles(self):
         self.game_play_page.avoid_obstacles(5)
-        #check that player has 2 or 3 lives left (sometimes 1 life is lost right when stopping the tests)
-        assert_that(self.game_play_page.get_current_life()).described_as("player current life").is_greater_than(0) 
-
+    # check that player has 2 or 3 lives left (sometimes 1 life is lost right when stopping the tests)
```
Comment (robert-altom) [inline tests/game_play_test.py]: This comment should be indented at the same level as the code.

File: requirements.txt [hunk 7]
```diff
@@ -1,9 +1,9 @@
 pytest
```
Comment (robert-altom) [inline requirements.txt:1]: We don't use `pytest-html`, `assertpy`, `selenium`, `pytest-selenium`, or `xmlrunner` so you can remove them.

**Comment by SandyEma9** [comment]
@rucindrea @robert-altom

**Comment by SandyEma9** [comment]
image

**Review by rucindrea — APPROVED** [review]

**Pull request merged**